lib/between_meals/repo.rb

```diff
@@ -50,89 +50,89 @@ def self.get(type, repo_path, logger)
     end
 
     def exists?
-      fail 'Not implemented'
+      fail "#{__method__} not implemented"
```

Comment:
The fail message will include the function that threw the function, so this is redundant, no?